Assets/Scripts/Ammunition.cs

```diff
@@ -0,0 +1,57 @@
+using UnityEngine;
+
+public class Ammunition : MonoBehaviour
+{
+    public Transform hand;
```

Comment:
This `hand` reference seems to only be used as a boolean. Suggest replacing it with `bool isPickedUp` instead.